mod/developers/views/default/theme_sandbox/forms.php

```diff
@@ -103,6 +103,32 @@
 			'label' => 'Date input (.elgg-input-date):',
 		));
 
+		echo elgg_view('elements/forms/field', array(
+			'input' => elgg_format_element('div', array(
+				'class' => 'elgg-input-date',
+			)),
+			'label' => elgg_view('elements/forms/label', array(
+				'label' => 'Inine datepicker (div.elgg-input-date)',
```

Comment:
"Inline". I'm not sure we should encourage devs to duplicate this usage (no input element). And without the `data-` attribute it causes the deprecation notice.